tests/system/conftest.py

```diff
@@ -235,12 +237,12 @@ def file_data():
     return _file_data
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
 def kms_bucket_name():
     return _helpers.unique_name("gcp-systest-kms")
 
 
-@pytest.fixture(scope="session")
```

Comment:
Thanks for doing this. Do the tests pass if we leave this as "session"?

Reply:
It does, however the kms_bucket is modified in tests, so I think it's best that we change it to function